src/components/Upload/Details.tsx

```diff
@@ -183,25 +183,6 @@ const Details: FC<Props> = ({ onUpload, onCancel }) => {
               />
             </div>
           </div>
```

Comment:
Make sure `acceptTerms` property in the form initialization and default values were removed.

Reply:
Ok will look into this and try to fix this.

Reply:
Have made the change as suggested by you. Is it fine now or I am missing something?